Example/Progress+UIViewController.swift

```diff
@@ -71,4 +80,19 @@ extension UIViewController {
             self.present(alert, animated: true, completion: nil)
         }
     }
+
+    /// Method to add a Attributed string as Error message for a given UIAlertController
+    /// - Parameters:
+    ///   - alertController: UIAlertController object in which you want to display error message
+    ///   - message: Error message description
+    func displayErrorInAlertController(alertController: UIAlertController, message: String) {
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = NSTextAlignment.center
+        let attributedString = NSAttributedString(string: message, attributes: [
+            NSAttributedString.Key.paragraphStyle: paragraphStyle,
+            NSAttributedString.Key.font: UIFont.systemFont(ofSize: 14),
+            NSAttributedString.Key.foregroundColor: UIColor.red
+        ])
+        alertController.setValue(attributedString, forKey: "attributedMessage")
```

Comment:
Using Apple private API?
Hmm. I'm not sure - maybe it's ok for the sample app that will never be released.

Reply:
Yeah, I was thinking to create a Custom Alert button but thought of using private API to make the job easier for the sample app.

Reply:
I couldn't agree more with @donnie-jp, especially if it's just to have a custom colored message blinking in the alert :)
But I have to admit it is nicely done and Apple will never review this app, so... Let's just hope Apple don't refactor its AlertController in a near future. ;)

Reply:
Totally agreed. Let me if I can add some other alternative approach

Reply:
I guess the other alternatives also end up in modifying UIAlertController(Which we shouldn't). So I guess we can go ahead with this solution

Reply:
Yes, the main purpose of this PR is to filter bad UUID, which is done properly. We can merge this PR and change the controller if something goes wrong in our future tests. A simple change would be to have a regular UIViewController instead of an UIAlertViewController